_episodes/first_example_recipe.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "Running a recipe (First example)"
+teaching: 20
+exercises: 25
+questions:
+- "What is a recipe?"
+- "How can I do the same preprocessing on many different datasets?"
+- "What are the files and directories that are created after running a recipe?"
+- "What happens when I run a recipe?"
+objectives:
+- "Run an ESMValTool recipe"
+- "Understand the purpose of different settings in the recipe"
+- "Inspect the output directories"
+- "Examine the log information"
+keypoints:
+- "A recipe does not break by fiddling with it"
+- "Log information is useful when interpreting the first warnings/errors"
+- "The dataset section resembles the filename or directory structure (e.g. CMIP subdirectories)"
+---
+
+This episode describes how ESMValTool recipes work, how to run a recipe and how to explore the recipe output. By the end of this episode, you should be able to run your first recipe, look at the recipe output, modify a recipe, explore and run some basic recipe debugging.
+
+## Introduction to Recipes
+
+Recipes are the instructions that you give to ESMValTool that tell it what you want to do. This includes four main sections: datasets, preprocessors, diagnostics and description.
+
+  - datasets: what datasets you want to use, including
+    - the time range and time resolution,
```

Comment:
"time period" instead of "time range"?
"temporal resolution" instead of "time resolution"?